svthalia/concrexit
# Add the option to disable Thalia pay for events

Closes #1422 



### Summary
You can now disable Thalia Pay for specififc events

### How to test
1. Create an event with a price and uncheck `Is Thalia Pay payment allowed for this event`
2. Make sure you have Thalia Pay enabled
3. Notice that there is no longer a `Pay with Thalia Pay` button on the event page
